### DISCUSSION TITLE
Compatible Chart controls for blazor fluent ui

### DISCUSSION THREAD

**Discussion by satyajit-behera** [microsoft/fluentui-blazor#1888] [Q&A]
Hi, Can anyone please help me with any good chart control available for blazor fluent ui and compatible with the design, easy to integrate. Thanks.

**Comment by JohnVottero** [comment]
I'm just getting started with Blazor-ApexCharts but, so far, it is working really well.

https://github.com/apexcharts/Blazor-ApexCharts

**Comment by AJ1000** [comment]
Try out https://puresourcecode.com/dotnet/blazor/blazor-component-for-chartjs/
